.github/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 ### Improvements
 
+* Changing deprecated abstract property definitions
+  in _device.py.
```

Comment:
(since the underscore means italics in markdown)
```suggestion
  in `_device.py`.
```

Reply:
Fair point, thanks!